guardian/discussion-rendering
# Prettier updates

## What?
This PR updates the prettier config to:

- Use `@guardian/prettier` configs
- Use the latest `prettier` version
- Check with `yarn prettier:check`
- Fix with `yarn prettier:fix`
- Check code for prettier errors before pushing
- Runs prettier over the whole codebase

## Why?
To bring this repos prettier config up to date and prevent code being committed that does not follow the standard formatting rules.
